app/javascript/react/src/App.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'; 
+import Greeting from './components/Greeting';
+
+function App() {
+  return (
+    <Greeting />
+  );
```

Comment:
- [ ] according to the requirements the `Greeting` component should be setup as a route in the app component.

Reply:
OOps.. 😄 I beleive I missed that, let me quickly add `react-router` and set it up for you

Reply:
@mikethreels thank you for your feedback I have made the required changes 🚀